docs/components/MDX/componentMapping.tsx

```diff
@@ -17,12 +19,14 @@ const componentMapping = {
   p: (props: any) => <Paragraph marginBottom={majorScale(3)} {...props} />,
   strong: (props: any) => <Strong {...props} />,
   ol: (props: any) => <Ol {...props} />,
-  ul: (props: any) => <Ul {...props} />,
+  ul: (props: any) => <Ul {...props} marginTop="-16px" />,
```

Comment:
Do we want this to be universally true? Can we see a screenshot of what this change looks like?

Reply:
Yes for now. We'll likely have to revisit the margin for headings and paragraphs because this is just a patch.

With this change:
![image](https://user-images.githubusercontent.com/679627/128932574-d1d10da4-15c8-4fc1-97c5-821277f54b7a.png)

Without:
![image](https://user-images.githubusercontent.com/679627/128932630-3513282b-8279-4140-add3-1e2f0d2e950d.png)